Gemfile.tt

```diff
@@ -53,7 +53,7 @@ group :development, :test do
   # Code Analysis
   gem 'bullet' # help to kill N+1 queries and unused eager loading
   gem 'brakeman', require: false # A static analysis security vulnerability scanner for Ruby on Rails applications
-  gem 'parser', '3.0.1.1' # Use correct parser version to avoid parser warnings
+  gem 'parser', '3.2.2.3' # Use correct parser version to avoid parser warnings
```

Comment:
Do we know why we need this gem?

Reply:
I didn't have time to check that as it already in the template @olivierobert there are some gem that I think we don't really use it at all but still in the Gemfile.tt, I will create a separate issue to revisit the Template Gemfile.tt 👌

Reply:
Issue created https://github.com/nimblehq/rails-templates/issues/426